### PR TITLE
Add CSE Hiring Exercise based on Grouping Exercise

### PR DESCRIPTION
**Pull request by richardtru**
Adding clarity and removing ambiguity to the backend exercise for more structured submissions.

### CODE REVIEW

**Review by zzidante — APPROVED** [review]
Co-created with Rich.

File: support/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Programming Exercise - Support Engineer
+
+The goal of this exercise is to identify rows in a CSV file that
+__may__ represent the __same person__ based on a provided __Matching Type__ (definition below).
```
Comment (Alan-Marx) [inline support/README.md:4]: I find the `__may__`, `__same__`, and `__Matching Type__` expressions more confusing than their standard English renderings. Makes me think of Ruby keywords such as `__dir__` and `__FILE__`.

Comment (richardtru) [reply support/README.md:4]: @Alan-Marx it's markdown syntax. Please see here: https://github.com/retailzipline/hiring-exercises/tree/b5232e9bf423804424d7d74185de4df2a582d1a6/support

**Comment by Alan-Marx** [comment]
@zzidante, @richardtru - Gave my two cents

**Comment by richardtru** [comment]
@pablitozip and @gavin-zipline have approved!

**Pull request merged**